magda-web-client/src/Components/Editing/Editors/multiItem.tsx

```diff
@@ -153,6 +166,7 @@ export class ListMultiItemEditor<V> extends MultiItemEditor<V> {
                         onChange={onChange}
                         canBeAdded={canBeAdded}
                         addOnChange={addOnChange}
+                        renderAbove={!!renderAbove}
```

Comment:
Because it's an optional parameter and could be `undefined`

Reply:
Shouldn't it be `renderAbove?: boolean` then?

Reply:
It is
https://github.com/magda-io/magda/pull/2642/files#diff-7eb9891f4774ad92392198dd3ec38b59

Reply:
It is there but not on line 153, I've left another comment there :)

Reply:
Oh I see. So `renderAbove: boolean = true` sets the default value for `renderAbove` to be true, so if nothing is passed in, it will be `true`, so in a way, doing the `!!` is unnecessary. Removed the conversion to boolean. Thanks for picking up on it.